.travis.yml

```diff
@@ -42,17 +40,31 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda install conda-build anaconda-client
   - conda update -q conda conda-build
-  - conda config --add channels $PYDM_CHANNEL 
   - conda config --append channels pcds-tag
   - conda config --append channels conda-forge
   # Useful for debugging any issues with conda
   - conda info -a
-  # Grab all dependencies
-  - conda build -q conda-recipe --python=$TRAVIS_PYTHON_VERSION --output-folder bld-dir
-  - conda config --add channels "file://`pwd`/bld-dir"
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION typhon --file dev-requirements.txt
-  # Launch Conda environment
-  - source activate test-environment
+  - |
+    # Install all dependencies via CONDA
+    if [[ $PYDM_CHANNEL != "None" ]]; then
+      conda config --add channels $PYDM_CHANNEL
+      # Use the recipe contained inside the repository to build packcage
+      conda build -q conda-recipe --python=$TRAVIS_PYTHON_VERSION --output-folder bld-dir
+      # Install package with development dependencies
+      conda config --add channels "file://`pwd`/bld-dir"
+      conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION typhon --file dev-requirements.txt
+      # Launch Conda environment
+      source activate test-environment
```

Comment:
At some point you may consider switching to `conda activate` and setting up the conda environment in the new way.

Reply:
Is it as simple as just switch the one word? In that case I'll do it rn